### PR TITLE
Chapter 10

### PR DESCRIPTION
**Pull request by balqis277**
ready

### CODE REVIEW

**Comment by balqis277** [comment]
ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -27,59 +35,67 @@ textarea {
...
 }
+
 input {
   height: auto !important;
 }
-  
+
 input.btn.btn-primary {
-  margin-top: 10px;
+    margin-top: 10px;
```
Comment (anlvq-1058) [inline app/assets/stylesheets/custom.scss:96]: wrong indent

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -1,30 +1,77 @@
...
-    @user = User.new(user_params) # Not the final implementation!
+    @user = User.new(user_params)
     if @user.save
-      flash[:success] = "Welcome to the Sample App!"
+      flash[:success] = "Welcome to Sample App!"
       redirect_to @user
-    # Handle a successful save.
     else
-      render :new
+      render "new"
```
Comment (anlvq-1058) [inline app/controllers/users_controller.rb:24]:
```suggestion
      render :new
```

File: app/views/shared/_error_messages.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+      <div class="alert alert-danger">
+        The form contains
+        <%= pluralize(@user.errors.count, "error") %>.
+      </div>
+      <ul>
+      <% @user.errors.full_messages.each do |msg| %>
+        <li><%= msg %></li>
+      <% end %>
+      </ul>
+    </div>
```
Comment (anlvq-1058) [inline app/views/shared/_error_messages.html.erb:12]: wrong indent

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by balqis277** [comment]
ready

**Comment by balqis277** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**